File: problems/3168-minimum-number-of-chairs-in-a-waiting-room/solution.py
class Solution:
    def minimumChairs(self, s: str) -> int:
        chairs = 0
        max_chairs = 0
        for char in s:
            if char == 'E':
                chairs += 1
                max_chairs = max(max_chairs, chairs)
            else:
                chairs -= 1
        return max_chairs

File: problems/3168-minimum-number-of-chairs-in-a-waiting-room/solution.c
int minimumChairs(char* s) {
    int chairs = 0;
    int maxChairs = 0;
    for (int i = 0; s[i] != '\0'; i++) {
        if (s[i] == 'E') {
            chairs++;
            if (chairs > maxChairs) {
                maxChairs = chairs;
            }
        } else {
            chairs--;
        }
    }
    return maxChairs;
}